new-harts-rules-the-oxford-style-guide.csl

```diff
@@ -372,17 +383,24 @@
       </choose>
     </group>
     <choose>
-      <if variable="URL">
-        <text variable="URL" prefix=" &lt;" suffix="&gt;"/>
-        <group prefix=" [" suffix="]">
-          <text term="accessed"/>
-          <date variable="accessed">
-            <date-part name="day" prefix=" "/>
-            <date-part name="month" prefix=" "/>
-            <date-part name="year" prefix=" "/>
-          </date>
-        </group>
+      <if variable="DOI">
+        <text variable="DOI" prefix=". doi:"/>
```

Comment:
Can you please move the ". " to be a delimiter on a group containing these items, rather than using prefix/suffix for punctuation? This will prevent extra punctuation from being added when a variable is missing. 

Please do this for all do the delimiters between elements in this macro